include/superior_mysqlpp/low_level/dbdriver.hpp

```diff
@@ -105,6 +105,8 @@ namespace SuperiorMySqlpp { namespace LowLevel
         /** Logger instance pointer. */
         Loggers::SharedPointer_t loggerPtr;
 
+        unsigned long clientFlags = 0;
```

Comment:
I think we should aim for a solution that doesn't require storing data here that are required only during initialization.